docs/standard/base-types/common-type-system.md

```diff
@@ -88,7 +88,7 @@ The common type system defines how types are declared, used, and managed in the
   
 <a name="Structures"></a>   
 ### Structures  
- A structure is a value type that derives implicitly from <xref:System.ValueType?displayProperty=fullName>, which in turn is derived from <xref:System.Object?displayProperty=fullName>. A structure is very useful for representing values whose memory requirements are small, and for passing values as by-value parameters to methods that have strongly typed parameters. In the .NET Framework class library, all primitive data types (<xref:System.Boolean>, <xref:System.Byte>, <xref:System.Char>, <xref:System.DateTime>, <xref:System.Decimal>, <xref:System.Double>, <xref:System.Int16>, <xref:System.Int32>, <xref:System.Int64>, <xref:System.SByte>, <xref:System.Single>, <xref:System.UInt16>, <xref:System.UInt32>, and <xref:System.UInt64>) are defined as structures.  
+ A structure is a value type that derives implicitly from <xref:System.ValueType?displayProperty=fullName>, which in turn is derived from <xref:System.Object?displayProperty=fullName>. A structure is very useful for representing values whose memory requirements are small, and for passing values as by-value parameters to methods that have strongly typed parameters. In .NET class library, all primitive data types (<xref:System.Boolean>, <xref:System.Byte>, <xref:System.Char>, <xref:System.DateTime>, <xref:System.Decimal>, <xref:System.Double>, <xref:System.Int16>, <xref:System.Int32>, <xref:System.Int64>, <xref:System.SByte>, <xref:System.Single>, <xref:System.UInt16>, <xref:System.UInt32>, and <xref:System.UInt64>) are defined as structures.  
```

Comment:
> In .NET class library, ...

In .NET class libraries, ...

or

In a .NET class library, ...

... and the comma splice in the previous sentence almost knocked me right out of my chair. :grinning: